build.sbt

```diff
@@ -18,14 +18,15 @@ val shared = Seq(
     )
   ),
   libraryDependencies ++= Seq(
-    "com.lihaoyi" %%% "utest" % "0.3.0" % "test"
+    "com.lihaoyi" %%% "utest" % "0.3.1" % "test"
   ),
   scalaJSStage in Global := FullOptStage,
   organization := "com.lihaoyi",
   version := _root_.fastparse.Constants.version,
-  scalaVersion := "2.11.6",
-  libraryDependencies += "com.lihaoyi" %% "acyclic" % "0.1.2" % "provided",
-  addCompilerPlugin("com.lihaoyi" %% "acyclic" % "0.1.2"),
+  scalaVersion := "2.11.7",
+  ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) },
```

Comment:
it was a suggestion of sbt when it complained for several different versions of Scala being used by the modules. I just did a clean build without this line and the warning does not appear anymore.